src/ui/ManageCookiesDialog/Section.svelte

```diff
@@ -1,17 +1,19 @@
 <script lang="ts">
   import Svg from '@/ui/Svg/svelte'
+  import { IsMobile } from '@/stores/responsive'
 
   export let title: string
   export let description: string
 
   let isOpened = false
 </script>
 
-<section>
+<section class:body-2={$IsMobile}>
```

Comment:
Redundant due to https://github.com/santiment/san-webkit/pull/61/files#diff-4e6ba6a2d44390122a5c90bdb03f8ed41eb707557b636c5981de2c05e29b3aabR60